docs/containers/testing-with-docker.md

```diff
@@ -0,0 +1,58 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Testing with docker](#testing-with-docker)
+  - [Using the included docker-compose support](#using-the-included-docker-compose-support)
+    - [Usage](#usage)
+    - [Configuration](#configuration)
+    - [Notes](#notes)
+    - [Example](#example)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Testing with docker
+
+## Using the included docker-compose support
+
+### Usage
+
+```bash
+docker build -t lyft/cartography
+docker-compose up -d
+docker-compose run cartography ...
```

Comment:
Isn't cartography already started with the command above?

Reply:
No, it's `docker-compose run cartography cartography ...`